addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -593,6 +672,9 @@ export class OdooEditor extends EventTarget {
         clearInterval(this._snapshotInterval);
         this._pluginCall('destroy', []);
         this.isDestroyed = true;
+        // Remove table UI
+        this._rowUi.remove();
+        this._columnUi.remove();
```

Comment:
Can you have both at the same time ? I suspect no. Then can't we have a system that prevents us to have both and so we just have to clean one variable on this and not two ?

Reply:
Yes you can (and really should). If you're in the first cell of the first row.